### DISCUSSION TITLE
Question regarding the (MCA) tool

### DISCUSSION THREAD

**Discussion by iranianurbanist** [pysal/momepy#569] [Q&A]
Hello and thank you for providing this useful analysis package.

I was looking for the answer to my question regarding how to use (MCA) in the previous discussions, but unfortunately I had to ask my question again. In one of the discussions, for using shapefiles, an answer was given that referred to the (Converting from GeoDataFrame to Graph and back) topic. But I must admit honestly that despite being aware of the backbone and structure of this code, I do not know how to call my file and convert it to a "Graph".

Then where should I call the final data so that I can use it for the (Multiple Centrality Assessment) parameters?

Please guide me in this regard.

Thank you for your help.

**Comment by martinfleis** [comment]
I am really not sure I understand the question, sorry. 

> how to call my file and convert it to a "Graph"

That is documented here http://docs.momepy.org/en/stable/user_guide/graph/convert.html

> Then where should I call the final data so that I can use it for the (Multiple Centrality Assessment) parameters?

No idea what you mean by this question, sorry.

**Reply by iranianurbanist** [reply]
Hello again
First, thank you for your response and attention to users' questions.
You answered my first question about converting a file to a graph, but if I want to ask my second question in a simpler way, I should say, "How can I integrate the method (http://docs.momepy.org/en/stable/user_guide/graph/convert.html) with the (MCA) method to analyze the centrality of my shapefile?"

**Reply by martinfleis** [reply]
First convert your data to the graph and then follow http://docs.momepy.org/en/stable/user_guide/graph/centrality.html#Primal-graph
